data/json/items/armor/coats.json

```diff
@@ -15,10 +15,10 @@
     "symbol": "[",
     "looks_like": "coat_winter",
     "color": "yellow",
-    "covers": [ "torso", "arm_l", "arm_r" ],
-    "coverage": 100,
-    "encumbrance": 22,
-    "max_encumbrance": 30,
+    "armor_portion_data": [
+      { "covers": [ "torso" ], "coverage": 100, "encumbrance": [ 22, 26 ] },
```

Comment:
Please mention all cases like this where you change the max encumbrance to a different value than previous.

Reply:
In what way? A separate list in comment?

Reply:
Or in the opening post, yes.